common/scala/src/main/scala/whisk/core/containerpool/logging/DockerToActivationFileLogStore.scala

```diff
@@ -75,6 +85,8 @@ class DockerToActivationFileLogStore(system: ActorSystem, destinationDirectory:
    * once the defined limit is reached.
    */
   val bufferSize = 100.MB
+  val perms = java.util.EnumSet.of(OWNER_READ, OWNER_WRITE, GROUP_READ, GROUP_WRITE, OTHERS_READ, OTHERS_WRITE)
+  val attr = PosixFilePermissions.asFileAttribute(perms)
```

Comment:
Is there a more restrictive set of permissions?

Reply:
The permissions in the current code base are too restrictive. Perhaps the `OTHERS_WRITE` can be removed here though.

Reply:
Permissions from current code base are `OWNER_READ`, `OWNER_WRITE`, `GROUP_READ `, `OTHERS_READ`. We need `GROUP_WRITE` for a log issue.